src/engines/aurora/console.cpp

```diff
@@ -57,6 +57,11 @@
 #include "src/engines/aurora/console.h"
 #include "src/engines/aurora/util.h"
 
+#include "src/graphics/mesh/meshman.h"
+#include "src/graphics/shader/surfaceman.h"
+#include "src/graphics/shader/materialman.h"
+#include "glm/gtc/matrix_transform.hpp"
```

Comment:
Could you move the glm include between the boost and common includes, please? Separated with a new line in both directions.